data/json/monsters/reptile_amphibian.json

```diff
@@ -416,6 +416,160 @@
+  },
+  {
+    "id": "mon_turtle_big_s",
+    "type": "MONSTER",
+    "name": { "str_sp": "dement's mirror hatchling" },
+    "description": "A large, mostly aquatic mutated freshwater turtle having recently hatched, its shell already developing abnormal growths, grooves and reflections.",
+    "copy-from": "mon_turtle_s",
+    "default_faction": "mutant",
+    "volume": "750 ml",
+    "weight": "1 kg",
+    "hp": 18,
+    "speed": 15,
+    "dodge": 0,
+    "material": [ "flesh" ],
+    "symbol": "t",
+    "color": "green",
+    "aggression": -30,
+    "upgrades": { "age_grow": 250, "into": "mon_turtle_big" },
+    "fear_triggers": [ "PLAYER_CLOSE", "SOUND" ],
+    "harvest": "mammal_tiny",
+    "flags": [ "SEES", "HEARS", "SMELLS", "KEENNOSE", "CLIMBS", "PATH_AVOID_DANGER_1", "ANIMAL", "SWIMS" ],
+    "armor": { "bash": 5, "cut": 5, "stab": 5, "heat": 2, "bullet": 4, "electric": 1, "acid": 1 }
+  },
+  {
+    "id": "mon_turtle_giant",
+    "type": "MONSTER",
+    "name": { "str": "lunatic's vision turtle" },
+    "description": "A massive, mostly aquatic freshwater turtle, about six times larger than the norm, ponderously moving about its obviously mutated and bulging shell exhibiting strange, almost hypnotic patterns and shapes filled with sharp edges. It probably doesn't consider you as prey, but be careful around its large head.",
+    "copy-from": "mon_turtle_big",
+    "proportional": { "hp": 4, "speed": 1, "morale": 1.43, "melee_dice_sides": 1.5 },
+    "volume": "226 L",
+    "weight": "50 kg",
+    "aggression": -10,
+    "melee_skill": 5,
+    "melee_damage": [ { "damage_type": "cut", "amount": 7 } ],
+    "reproduction": { "baby_monster": "mon_turtle_s", "baby_count": 13, "baby_timer": 240 },
+    "baby_flags": [ "AUTUMN", "SUMMER" ],
+    "upgrades": { "half_life": 65, "into": "mon_turtle_mega" },
+    "zombify_into": "mon_meat_cocoon_med",
+    "armor": { "bash": 15, "cut": 15, "stab": 15, "heat": 5, "bullet": 8, "electric": 3, "acid": 3 }
+  },
+  {
+    "id": "mon_turtle_mega",
+    "type": "MONSTER",
+    "name": { "str": "madman's maze turtle" },
+    "description": "The shell of this enormous, mostly aquatic mutated turtle hurts to look at, filled with sharp edges, unnatural shapes and eye-bending reflections, the whole hosting its own ecosystem of aquatic fauna and flora. The creature itself moves at a crawling pace, but its hungry maw looks ready to snatch up any morsel within reach.",
+    "default_faction": "mutant",
+    "bodytype": "turtle",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "REPTILE" ],
+    "diff": 10,
+    "volume": "620 L",
+    "weight": "760 kg",
+    "hp": 600,
+    "speed": 10,
+    "material": [ "flesh" ],
+    "symbol": "T",
+    "color": "green",
+    "aggression": 40,
+    "morale": 80,
+    "aggro_character": false,
+    "melee_skill": 7,
+    "melee_dice": 3,
+    "melee_dice_sides": 6,
+    "melee_damage": [ { "damage_type": "cut", "amount": 8 } ],
```

Comment:
If you want them to bite, add a bite attack - I'd suggest a grabbing bite + drag.

Reply:
Will do, those turtles I don't want focused on their bites, but they would have one, how would I go about them being able to slam you with a spiked shell?